src/packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -573,14 +594,15 @@ describe('sqlite', () => {
 
     await expect(result).rejects.toMatchInlineSnapshot(`
 
-                                                                                                                                                                                                                                                                                    ⚠️ We found changes that cannot be executed:
+                                                                                                                                                                                                                                                                                                ⚠️ We found changes that cannot be executed:
```

Comment:
Is there meant to be this much whitespace?

Reply:
White space in snapshot is a bit out of control, if you know how to make them behave it's something we can improve in general in tests.